src/components/onboarding.tsx

```diff
@@ -75,20 +75,16 @@ export default function Onboarding() {
 		},
 	} );
 
-	useEffect( () => {
-		const run = async () => {
-			const { path, name, isWordPress } = await getIpcApi().generateProposedSitePath(
-				generateSiteName( [] )
-			);
-			setSiteName( name );
-			setProposedSitePath( path );
-			setSitePath( '' );
-			setError( '' );
-			setDoesPathContainWordPress( isWordPress );
-		};
-		run();
-		// eslint-disable-next-line react-hooks/exhaustive-deps
-	}, [] );
+	const run = async () => {
+		const siteName = await generateSiteName( [] );
+		const { path, name, isWordPress } = await getIpcApi().generateProposedSitePath( siteName );
+		setSiteName( name );
+		setProposedSitePath( path );
+		setSitePath( '' );
+		setError( '' );
+		setDoesPathContainWordPress( isWordPress );
+	};
+	run();
```

Comment:
I don't understand this change. Why should we move this logic outside the `useEffect` callback? 🤔

Reply:
We shouldn't 😅 
I rolled back the change.